pkg/kubelet/cm/cpumanager/policy_options.go

```diff
@@ -138,6 +148,10 @@ func NewStaticPolicyOptions(policyOptions map[string]string) (StaticPolicyOption
 		return opts, fmt.Errorf("static policy options %s and %s can not be used at the same time", DistributeCPUsAcrossNUMAOption, DistributeCPUsAcrossCoresOption)
 	}
 
+	if opts.AlignByUnCoreCacheOption && opts.DistributeCPUsAcrossCores {
```

Comment:
IIUC we are always compatible with `AlignBySocket` considering how available hardware (x86, arm) is engineered, correct?